adafruit_si7021.py

```diff
@@ -141,3 +143,66 @@ def start_measurement(self, what):
         elif self._measurement != what:
             raise RuntimeError("other measurement in progress")
         self._measurement = what
+
+    @property
+    def serial_number(self):
+        """The device's unique ID (serial number)."""
+        serial, _ = self._get_device_info()
```

Comment:
return self._get_device_info()[0]

Reply:
Thanks, fixed in 6587e8749b9aba6e66101219cd3dd706613214f7